hack/demo.sh

```diff
@@ -48,16 +48,21 @@ spec:
   configuration:
     exclude:
     - not_useful
-    - test:conftest-clair
+    include:
+    - slsa_provenance_available
   sources:
-    - name: Default EC policy
-      policy:
-      - quay.io/hacbs-contract/ec-release-policy:latest
+  - data:
+    - github.com/hacbs-contract/ec-policies.git//data
+    name: ec-demo
+    policy:
+      - github.com/hacbs-contract/ec-policies.git//policy/lib
+      - github.com/hacbs-contract/ec-policies.git//policy/release
```

Comment:
I think it's better to keep using the quay.io url. That's what we're moving towards in other places. That bundle includes the lib dir, so you only need the one url.

Reply:
change back to quay.io/hacbs-contract/ec-release-policy:latest